src/main/java/pl/nogacz/snake/pawn/PawnClass.java

```diff
@@ -13,22 +13,41 @@
     public PawnClass(Pawn pawn) {
         this.pawn = pawn;
     }
-
     public ImageView getImage() {
         Image image = new Image(Resources.getPath(pawn + ".png"));
         return new ImageView(image);
     }
 
     public ImageView getImageDirection(int direction) {
         String direct = "";
-
-        switch(direction) {
-            case 1: direct = "UP"; break;
-            case 2: direct = "BOTTOM"; break;
-            case 3: direct = "LEFT"; break;
-            case 4: direct = "RIGHT"; break;
+        SnakeSkins.bodySkinTones myVar = SnakeSkins.getBodySkin();
+        if(myVar == SnakeSkins.bodySkinTones.SKIN_TONE_1) {
+            switch(direction) {
+                case 1: direct = "UP"; break;
+                case 2: direct = "BOTTOM"; break;
+                case 3: direct = "LEFT"; break;
+                case 4: direct = "RIGHT"; break;
+                default: break;
+            }
+        }
+        else if(myVar == SnakeSkins.bodySkinTones.SKIN_TONE_2) {
+            switch(direction) {
+                case 1: direct = "UP_SKIN_2"; break;

```

Comment:
Bunları bir map e koyabilir misin?

skin tone, direction => skin

MultiKeyMap kullanabilirsin:
https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/map/MultiKeyMap.html

O zaman bu if else bloğuna ihtiyaç kalmaz. Yalnızca tek bir satırda map i direction ve skin tone ile çağırırsın.

Reply:
Hocam dediğinizi yapmayı denedim ancak benim skin tone enum, direction int ve skin String olduğu için yapamıyorum.
<K,V> den K'yı enum type tanımladığım zaman Key1 ve Key2 de enum oluyor. (Oysa Key2 burada String olmalıydı)